src/ttsim/main_target.py

```diff
@@ -112,11 +123,12 @@ class Targets(MainTargetABC):
 
 @dataclass(frozen=True)
 class Labels(MainTargetABC):
+    all_qnames_in_policy_environment: str = "labels__all_qnames_in_policy_environment"
```

Comment:
Please move this and `policy_inputs` down in the list, these are among the least important / purely internal ones. Instead, `input_columns` should move up now that it might actually be used.